script/sync_package_json.clj

```diff
@@ -1,19 +1,17 @@
 (ns sync-package-json
-  (:require [clojure.edn :as edn]
-            [cheshire.core :as json]))
+  (:require [jsonista.core :as j]))
```

Comment:
Tiny nitpick, but I think single letter and heavily abbreviated namespace aliases should be avoided. `json` carries way more information than `j`. Those single letter aliases always mean multiple scrolls back to the top of the file to figure out what that namespace does. I will make an exception for extremely common libraries with well established aliases like `str` for `clojure.string`, and `ig` for `integrant`.

Reply:
@zonotope Addressed this in a new commit FYI.